src/App.js

```diff
@@ -1,19 +1,45 @@
 import React from 'react';
 import './App.css';
-
+import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import { Container } from 'react-bootstrap';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoForm } from './components/TodoForm/TodoForm';
 import users from './api/users';
+import todos from './api/todos';
+
+const prepearedtodos = todos.map(todo => ({
+  ...todo,
+  userName: users.find(user => user.id === todo.userId).name,
+}));
+
+class App extends React.Component {
+  state = {
+    todosOnPage: prepearedtodos,
+  }
+
+  addTodo = ({ todoTitle, userName }) => {
+    this.setState(prevState => ({
+      todosOnPage: [...prevState.todosOnPage, {
+        completed: false,
+        id: prevState.todosOnPage[prevState.todosOnPage.length - 1].id + 1,
+        title: todoTitle,
+        userName,
+        userId: users.find(user => user.name === userName).id,
```

Comment:
let's move this to variable newTodo and then use it in setState

this.setState(prevState => ({
      todos: [...prevState.todos, newTodo ])}